Felienne/hedy
# Remove null value displyed in log search form when there are no values

**Description**


**Fix for**

#1873

**How to test**

When deployed to alpha, log as admin and go to /admin/stats. Perform a search log and note that all anonymous logs have empty username instead of username `null`.
